### PR TITLE
Increase execution stack size

### PR DESCRIPTION
**Pull request by Leo-Besancon**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
  * [ ] if part of node-launch, checked using the `resync_check` flag
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

File: massa-execution-worker/src/worker.rs [hunk 1]
```diff
@@ -287,7 +287,9 @@ pub fn start_execution_worker(
 
     // launch the execution thread
     let input_data_clone = input_data.clone();
-    let thread_builder = thread::Builder::new().name("execution".into());
+    let thread_builder = thread::Builder::new()
+        .stack_size(200 * 1024 * 1024)
+        .name("execution".into()); //200 Mb stack size
```
Comment (sydhds) [inline massa-execution-worker/src/worker.rs:292]: Comment should be on the previous line

**Review by sydhds — COMMENTED** [review]
All good but I would have recommended to do the modification (stack size increase) then push another PR for the version updates

**Comment by Leo-Besancon** [comment]
> All good but I would have recommended to do the modification (stack size increase) then push another PR for the version updates

That's true, it would probably be better like that

**Comment by Leo-Besancon** [comment]
> All good but I would have recommended to do the modification (stack size increase) then push another PR for the version updates

The version update is done in https://github.com/massalabs/massa/pull/4730 instead of this one.

File: massa-execution-worker/src/worker.rs [hunk 2]
```diff
@@ -288,8 +288,8 @@ pub fn start_execution_worker(
     // launch the execution thread
     let input_data_clone = input_data.clone();
     let thread_builder = thread::Builder::new()
-        .stack_size(200 * 1024 * 1024)
-        .name("execution".into()); //200 Mb stack size
+        .stack_size(200 * 1024 * 1024) //200 Mb stack size
```
Comment (sydhds) [inline massa-execution-worker/src/worker.rs:291]: Sorry but I forgot to add that maybe a comment about why this size and that it is a temporary workaround would be nice :)

**Review by sydhds — APPROVED** [review]

**Pull request merged**